charts/px/templates/hooks/post-delete/px-postdelete-unlabelnode.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  namespace: kube-system
+  name: px-postdelete-unlabelnode
+  labels:
+    heritage: {{.Release.Service | quote }}
+    release: {{.Release.Name | quote }}
+    chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+  annotations:
+    "helm.sh/hook": post-delete
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  backoffLimit: 1
+  template:
+    spec:
+      serviceAccountName: tiller
+      restartPolicy: OnFailure
+      volumes:
+      - name: kubectl
+        hostPath:
+          path: /usr/bin/kubectl
```

Comment:
We are assuming `/usr/bin/kubectl` is present on the host path in the system. What is the guarantee?

Reply:
We could move it to `/opt/pwx` and call the command? That would be a safe bet since this is a delete hook and we could assume `/opt/pwx` is going to be there. I would fix this upon your comment @harsh-px

Reply:
My question is regarding the source location from where will you move it.

```
hostPath:
           path: /usr/bin/kubectl
```
means you are assuming that the host where this job runs will have a binary `/usr/bin/kubectl` at that location. What happens if it doesn't exist?